tests/conftest.py

```diff
@@ -31,6 +31,34 @@
 log = logging.getLogger(__name__)
 
 
+class OCSLogFormatter(logging.Formatter):
+
+    def __init__(self):
+        fmt = (
+            "%(asctime)s - %(levelname)s - %(name)s.%(funcName)s.%(lineno)d "
+            "- %(message)s"
+        )
+        super(OCSLogFormatter, self).__init__(fmt)
+
+
+def pytest_logger_config(logger_config):
+    logger_config.add_loggers([''], stdout_level='debug')
```

Comment:
Can we set this to 'INFO" and later enhance in customization to pass the debug as param. This will solve the huge logs issues actually which I was debugging and is basically due to this: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L376

Reply:
@vasukulkarni Sure, that's fixed in 6fa1295ec6fac2d6571a9fb1b389dd7a7c18046e

Reply:
thanks!